### PR TITLE
Claiming ownership of couple of tests

### PR DESCRIPTION
**Pull request by mnowaksuse**

### CODE REVIEW

File: tests/toolchain/gcc5_fortran_compilation.pm [hunk 1]
```diff
@@ -9,7 +9,7 @@
 # without any warranty.
 
 # G-Summary: Fortran test for SLE Toolchain Module's GCC5
-# G-Maintainer: Michal Nowak <mnowak@suse.com>
+# G-Maintainer: Michal Nowak <mnowak@suse.com>, Michal Nowak <mnowak@suse.com>
```
Comment (coolo) [inline tests/toolchain/gcc5_fortran_compilation.pm]: you *and* your twin? Please remove the G prefixes and just have one maintainer. it doesn't have to be perfect

**Comment by coolo** [comment]
if you could squash this, I'll merge

**Comment by sysrich** [comment]
@coolo have you played with the new Squash features baked into Github?

**Comment by coolo** [comment]
haeh? all I know of is that github now squashes the merge commit into the real commit.

**Pull request merged**